jupyterhub/app.py

```diff
@@ -2139,7 +2156,9 @@ def init_services(self):
             if orm_service is None:
                 # not found, create a new one
                 orm_service = orm.Service(name=name)
-                if spec.get('admin', False):
+                if spec.get(
+                    'admin', False
+                ):  # Todo: fix double assignment of admin roles
```

Comment:
I think we can leave this with a deprecation warning:
```suggestion
                ):
                    self.log.warning(
                        f"Service {name} sets `admin: True`, which is deprecated in JupyterHub 2.0."
                        " You can assign now assign roles via `JupyterHub.load_roles` configuration."
                    )
```

Reply:
Taken care of